### ISSUE TITLE
[Packages]There should be specific error/validation thrown while adding package which is already exits

### ISSUE THREAD

**Issue by spathare21** [sundeep-co-in/transtats#23]
##### ISSUE TYPE
 - Bug Report

##### OS / ENVIRONMENT
```
Fedora 25 x86 + Firefox 50. 
```
##### SUMMARY
- If we try to add package which is already exists it is not doing any validation or not any relevant error thrown

##### STEPS TO REPRODUCE
- Open transtats home page.
- Click on settings tab.
- Click on packages.
- You will see the list of packages.
- Click on Add Package.
- And add package which is already exits.
- Observe the behavior.

##### EXPECTED RESULTS
- There should be an proper error message after adding package which is already exits.
- Or while adding package which is already exits/added it have to validate and have to give error message.
- e.g one we add package in package name field it have to check that is it already exits. if yes then have to give proper message "Try to add new package/package is already exits/added"

##### ACTUAL RESULTS
- No any specific validation while adding duplicate package.
- and no any specific/relevant error after adding package which is already exits.

##### NOTE
- Attached screen capture which we got after trying to add duplicate package.

![package_error](https://cloud.githubusercontent.com/assets/23230884/21446898/916e9256-c8f2-11e6-8526-4cf3f2296bdb.png)

**Comment by sundeep-co-in** [comment]
Right. This would be resolving with **error handling** coming up in Transtats.

**Comment by juhp** [comment]
This is not just about adding duplicate packages?  A more generic problem, or is it?

**Comment by sundeep-co-in** [comment]
It's generic! In the sense, root cause is there is no `error handling` implemented. And this is one of the symptoms of that.

**Comment by krisnababu** [comment]
I have added memsource translation platform with invalid credentials. After that I tried to add CI Pipeline based on same TP, then I got errors/alerts (I guess one message is from server side validation)
 _Project details could not be fetched for the given URL.
 CI Pipeline will be associated with this project._

As I mentioned in the beginning, the issue is related to memsource's invalid login credentials and hence it would be helpful if error message gives information/alert about that!
